modules/swagger-core/src/test/java/io/swagger/v3/core/util/reflection/ReflectionUtilsTest.java

```diff
@@ -52,9 +71,11 @@ public void isOverriddenMethodTest() throws NoSuchMethodException {
 
         for (Method method : IParent.class.getMethods()) {
             if ("parametrizedMethod5".equals(method.getName())) {
```

Comment:
I think that there was a bug in the test. That method parametrizedMethod5 was hidden! and it should be listed.